includes/documents/class-wcpdf-invoice.php

```diff
@@ -311,33 +311,41 @@ public function init_settings() {
 				)
 			),
 			array(
-				'type'			=> 'setting',
-				'id'			=> 'number_format',
-				'title'			=> __( 'Number format', 'woocommerce-pdf-invoices-packing-slips' ),
-				'callback'		=> 'multiple_text_input',
-				'section'		=> 'invoice',
-				'args'			=> array(
-					'option_name'			=> $option_name,
-					'id'					=> 'number_format',
-					'fields'				=> array(
-						'prefix'			=> array(
-							'placeholder'	=> __( 'Prefix' , 'woocommerce-pdf-invoices-packing-slips' ),
-							'size'			=> 20,
-							'description'	=> __( 'to use the invoice year and/or month, use [invoice_year] or [invoice_month] respectively' , 'woocommerce-pdf-invoices-packing-slips' ),
+				'type'     => 'setting',
+				'id'       => 'number_format',
+				'title'    => __( 'Number format', 'woocommerce-pdf-invoices-packing-slips' ),
+				'callback' => 'multiple_text_input',
+				'section'  => 'invoice',
+				'args'     => array(
+					'option_name' => $option_name,
+					'id'          => 'number_format',
+					'fields'      => array(
+						'prefix'  => array(
+							'label'       => __( 'Prefix' , 'woocommerce-pdf-invoices-packing-slips' ),
+							'size'        => 20,
+							'description' => __( 'If set, this value will be used as number prefix.' , 'woocommerce-pdf-invoices-packing-slips' ) . ' ' .
+							/* translators: 1. document type, 2-3 placeholders */
+								sprintf( __( 'You can use the %1$s year and/or month with the %2$s or %3$s placeholders respectively.', 'woocommerce-pdf-invoices-packing-slips' ), __( 'invoice', 'woocommerce-pdf-invoices-packing-slips' ), '<strong>[invoice_year]</strong>', '<strong>[invoice_month]</strong>' ) . ' ' .
+								__( 'Check the Docs article below to see all the available placeholders for prefix/suffix.', 'woocommerce-pdf-invoices-packing-slips' ),
```

Comment:
I would break this in the `sprintf` commas instead, maybe for all placeholder values.